rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -156,18 +155,18 @@ SecRule REQUEST_HEADERS:Content-Length "!@rx ^\d+$" \
 
 #
 # Do not accept GET or HEAD requests with bodies
-# HTTP standard allows GET requests to have a body but this
-# feature is not used in real life. Attackers could try to force
-# a request body on an unsuspecting web applications.
+# In RCF-9110, "A client SHOULD NOT generate content in a HEAD/GET request 
+# unless it is made directly to an origin server that has previously indicated"
 #
 # -=[ Rule Logic ]=-
-# This is a chained rule that first checks the Request Method.  If it is a
-# GET or HEAD method, then it checks for the existence of a Content-Length
-# header.  If the header exists and its payload is either not a 0 digit or not
-# empty, then it will match.
+# The chained rule matches when:
+# 1) If the request method is GET or HEAD 
+# AND 
+# 2) Header: Content-Length is exists and non-zero
```

Comment:
```suggestion
# 2) Header: Content-Length exists and non-zero
```

Reply:
Fixed in [f779e7a](https://github.com/coreruleset/coreruleset/pull/3232/commits/f779e7a7a72362783de23524a5af95303db81d2f).